projects/blinking_led/src/main.c

```diff
@@ -0,0 +1,37 @@
+#include "interrupt.h"   // apparently soft timers needs this
+#include "log.h"         // similar to prinf but indicates the line #
+#include "soft_timer.h"  // soft timers
+#include "wait.h"        // for wait function
+
+#define COUNTER_PERIOD_MS 500  // the time between each count in ms
+
+typedef struct Counters {
+  uint8_t counter_a;
+  uint8_t counter_b;
+} Counters;
+
+void SoftTimerCallCounter(SoftTimerId timer_id, void *context) {
```

Comment:
The function should be static and also needs a prv in front of it as such
![image](https://user-images.githubusercontent.com/37023871/122568055-5abb9480-d017-11eb-890b-a175a0961f53.png)

Our coding standards are here:
https://uwmidsun.atlassian.net/wiki/spaces/ELEC/pages/9306218/Coding+Standards